server/boot/user-stats.js

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
+import { dayCount } from '../utils/date-utils';
```

Comment:
This file should be in server/utils as boot is reserved for function exports that require the server object at runtime.